network/jump.go

```diff
@@ -0,0 +1,65 @@
+// Copyright The Mantle Authors.
+// SPDX-License-Identifier: Apache-2.0
+package network
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// JumpDialer is a RetryDialer from a proxy.
+type JumpDialer struct {
```

Comment:
Might be a good idea to also add `var _ network.Dialer = (*JumpDialer)(nil)` (will need importing "github.com/flatcar-linux/mantle/network") to show in the code that this type is supposed to implement a certain interface.

Reply:
Fun, never seen this :)